src/util/uint256hash.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_UTIL_UINT256HASH_H
+#define BITCOIN_UTIL_UINT256HASH_H
+
+#include <uint256.h>
+
+#include <cstddef>
+#include <cstdint>
+
+class uint256Hash
```

Comment:
nit: `Uint256Hash` to abide to the naming convention.

Reply:
[Switched](https://github.com/bitcoin/bitcoin/pull/19374#issuecomment-650557986) to `SaltedTxidHasher` as [suggested](https://github.com/bitcoin/bitcoin/pull/19374#discussion_r445651273) by @jnewbery.